jenkins/hourly_maintenance.py

```diff
@@ -194,6 +195,16 @@ def DeleteCorruptGitRepos():
     return err
 
 
+def DeleteOldWorkspaces():
+    err = 0
+    for dirname in glob.glob('/var/lib/jenkins/workspace/*'):
```

Comment:
I guess we're confident the `/var/lib/jenkins/workspace/FOO` inode will get modified whenever we run the job?